oak/server/wasm_node.cc

```diff
@@ -227,6 +227,45 @@ void WasmNode::InitEnvironment(wabt::interp::Environment* env) {
       wabt::interp::FuncSignature(std::vector<wabt::Type>{wabtUsizeType, wabtUsizeType},
                                   std::vector<wabt::Type>{wabt::Type::I32}),
       this->OakRandomGet(env));
+
+  // These exported functions are used to run applications that have been ported to Wasm
+  // without code refactoring, and thus may require several WASI functions to be imported.
+  // WASI is an interface that provides access to OS features (e.g. filesystems, sockets, etc.).
+  // https://wasi.dev/
+  // These particular functions are required by TensorFlow Lite for Microcontrollers.
+  wabt::interp::HostModule* wasi_module = env->AppendHostModule("wasi_snapshot_preview1");
```

Comment:
Perhaps also add a link to https://github.com/CraneStation/wasi-libc/blob/12f5832b45c7450f8320db271334081247191d58/libc-bottom-half/headers/public/wasi/api.h#L1584 or something else to clarify where "wasi_snapshot_preview1" comes from, since I think it may change in the future, and it may be hard to figure out what to change it to.

Reply:
Done